python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -866,14 +866,14 @@ class RandomForestClassifier(Base):
         -----------
         params : dict of new params
         """
+        self.__init__()
```

Comment:
Shouldn't be necessary. __init__ was already called when this object was instantiated.

Reply:
I was getting an error "Cannot fit an already fitted RandomForest" if I didn't include this. This is the same format used in RandomForestRegressor.pyx